pkg/google/gke/gke.go

```diff
@@ -198,7 +214,15 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) error {
 
 				price, err := c.PricingMap.GetCostOfStorage(d.Region(), d.StorageClass())
 				if err != nil {
-					fmt.Printf("%s error getting cost of storage: %v\n", disk.Name, err)
+					c.logger.LogAttrs(ctx,
+						slog.LevelError,
+						err.Error(),
+						slog.String("disk_name", disk.Name),
+						slog.String("project", project),
+						slog.String("region", d.Region()),
+						slog.String("cluster_name", d.Cluster),
+						slog.String("storage_class", d.StorageClass()),
+					)
```

Comment:
This wasn't part of the issue linked in the PR, but it's good cleanup to convert the section to using slog.